test/e2e/performanceprofile/functests/2_performance_update/memorymanager.go

```diff
@@ -418,9 +435,15 @@ var _ = Describe("[rfe_id: 43186][memorymanager] Memorymanager feature", Label(s
 			workerRTNodes = getUpdatedNodes()
 			profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 			Expect(err).ToNot(HaveOccurred())
-			performanceMCP, err = mcps.GetByProfile(profile)
-			Expect(err).ToNot(HaveOccurred())
-
+			if !hypershift.IsHypershiftCluster() {
```

Comment:
I've noticed this code occurs multiple time over the tests, why not performing it once in a  higher `Describe` closure?